src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/EventHandlerModifier/Sources/InvalidHandlerModifier_Expected.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using PX.Data;
+
+namespace Acuminator.Tests.Sources
+{
+	public class MyExtension : ParentExtension
+	{
+		protected virtual void HandleCacheAttachedDiscDate(Events.CacheAttached<PX.Objects.AR.ARInvoice.discDate> e) // compiler doesn't identify polymorphism, wrong modifier will be suggested.
+		{
+			return;
+		}
+
+		protected virtual void HandleFieldUpdatedDiscDate(Events.FieldUpdated<PX.Objects.AR.ARInvoice.discDate> e)
+		{
+			return;
+		}
+
+		protected virtual void HandleCacheAttachedCuryDocBal(Events.CacheAttached<PX.Objects.AR.ARInvoice.curyDocBal> e)
```

Comment:
I would suggest adding unusual modifiers for one of event handlers. For example:
```cs
internal virtual protected  void HandleCacheAttachedCuryDocBal(Events.CacheAttached<PX.Objects.AR.ARInvoice.curyDocBal> e)
{

}
```

Reply:
Done. I also added a test with a "new" keyword.